iiblib/iib_client.py

```diff
@@ -157,6 +158,11 @@ def add_bundles(
                 "overwrite-from-index-token should be specified."
             )
 
+        if deprecation_list:
```

Comment:
You can probably condense this to

```
post_data['deprecation_list'] = []
if deprecation_list:
    post_data["deprecation_list"] = deprecation_list
```

Reply:
But then your code would add "deprecation_list" always to the post? Is it correct behavior that we want?

Reply:
oh I am commenting commit by commit ... I did not saw that she add it below.
Then I would suggest to change 
```
        post_data = {
            "from_index": index_image,
            "add_arches": arches,
           "deprecation_list": []
        }
```

end then keep the if part there. Or 

```
post_data["deprecation_list"] = deprecation_list if deprecation_list else list()
```
even tho the second one might not be that readable.

Reply:
@lipoja yeah, I thought of using your second example but it would be different from the whole script.  I think adding an empty list to post data is ok.

Reply:
done